app/src/main/java/com/infomaniak/mail/utils/LocalStorageUtils.kt

```diff
@@ -87,10 +97,16 @@ object LocalStorageUtils {
         return File(generateRootDir(context.attachmentsUploadRootDir, userId, mailboxId), draftLocalUuid)
     }
 
-    fun saveUploadAttachment(context: Context, uri: Uri, fileName: String, draftLocalUuid: String): File? {
+    fun saveUploadAttachment(
```

Comment:
`saveUploadAttachments`?

Reply:
Not like this, but yes it needs renaming.